docs/contributing.md

```diff
@@ -1,130 +1,61 @@
-# Collective Code Construction Contract - Oasis Implementation
+# Contributing
 
-The Collective Code Construction Contract (C4) is an evolution of the github.com [Fork + Pull Model](https://help.github.com/articles/about-pull-requests/), aimed at providing an optimal collaboration model for free software projects.
-This is the Oasis-specific implementation, based on [revision 2 of C4](https://github.com/zeromq/rfc/blob/63024673f19ad136652ff7b3bfb3a6547811e006/42/README.md).
+If you want to dive into the details, please see the [contract](./contract)
+that defines the contributor role in this project. If you're comfortable with
+a top-level summary, you can start here first.
 
-## Summary
+Our workflow is basically [GitHub Flow][github-flow] with specific roles:
 
-Thank you for contributing to Oasis! Here we try capture how we collaborate, and why we do it this way.
+- **Contributor:** Write patches that reduce the number of problems.
+- **Maintainers:** Merge patches that reduce the number of problems.
 
-This entire document is open for changes, if there is anything that is confusing or can be improved, please start a discussion with us!
+If you have an issue, it's best to open an issue to describe the problem and
+discuss solutions, but don't worry if you've already skipped that step.
 
-## License
+Assuming you already have a [developer install](./install.md) you should be
+able to start editing source code. There are a few useful commands you should
+know about:
 
-Copyright (c) 2009-2016 Pieter Hintjens.
+- **`npm install`**: Ensure that software dependencies are installed.
+- **`npm test`**: Ensure that all automated tests pass.
+- **`npm run fix`**: If an automated test failed, this may fix it.
 
-This Specification is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version.
+Please run `npm test` before writing a commit, because if there are errors then
+maintainers won't be able to merge your patch. Please ask for help if `npm test`
+is giving you any trouble.
 
-This Specification is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+## Frequently Failed Tests
 
-You should have received a copy of the GNU General Public License along with this program; if not, see <http://www.gnu.org/licenses>.
+### Unknown word
 
-## Abstract
+```
+/src/index.js:10:42 - Unknown word (Scuttlebtut)
```

Comment:
the build is failing because this is misspelled. Very meta, lol.

Reply:
Fixed! I've disabled the spell-checker for that line, but in the future another patch could:

- Replace the typo with a correctly-spelled word.
- Replace the typo with a word from `.cspell.json`. (I like this the most)